File: 13241/13241.py
# Backjoon Online Judge No.13241
# Python

# 범위가 늘어났지만 Python은 강력했다.
# 1934의 알고리즘을 그대로 이용하였다.

def gcd(x, y):
    while(y):
        x, y = y, x % y
    return x

A, B = map(int, input().split())

lcm = A * B // gcd(A, B)

print(lcm)

File: 13241/13241.c
// Backjoon Online Judge No.13241
// C

#include<stdio.h>

int gcd(int x, int y){
    while(y > 0){
        int tmp = y;
        y = x % y;
        x = tmp;
    }
    return x;
}

int main(void){
    // 일반적으로 int자료형의 범위는 약 2.1*10^9이다.
    // 입력값의 최대 범위는 10^8으로 int로 처리한다.
    int A = 0, B = 0;

    // 큰 수를 다룰 수 있도록 ull로 선언하였다.
    unsigned long long int lcm = 0;

    scanf("%d %d", &A, &B);
    lcm = (unsigned long long)A * B / gcd(A, B);
    
    printf("%llu\n", lcm);

    return 0;
}